common/dpsl.ts

```diff
@@ -355,6 +355,24 @@ export interface RunSmartctlCheckRequest {
   percentage_used_threshold?: number;
 }
 
+/**
+ * States of a diagnostics card used for setting UI
+ */
+export const enum DiagnosticsCardState {
```

Comment:
`enum` instead of `const enum`

https://google.github.io/styleguide/tsguide.html#enums

Reply:
Hi Yu-An,

This bug is being tracked in: b/301888904.

Currently there will be compile error if we use enum directly. I'll be in charge of doing the migration in a later time.